package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "core",
+  "name": "docs-core",
```

Comment:
shouldn't change this file. yeah, annoying it keeps changing..

Reply:
that's why i kept it in a separate commit ... i waited till this was the only pending thing to be sure it was not reintroduced by accident